test/test_api/test_call_signatures.py

```diff
@@ -122,10 +122,10 @@ def f(a, b):
 
 def test_get_signatures_whitespace(Script):
     s = dedent("""\
-    abs( 
+    abs(
```

Comment:
This change should not be removed. It's intentional. It's also why the tests are failing :)

Reply:
Ah.
I tend to have my editor strip trailing spaces (and spotted a few places where there were unexpected trailing spaces in jedi, which the `.editorconfig` file aims to help prevent), and suspect others might do too.

I realise it's a little less clear, but I wonder if this would be more robust if spelled as a single line string with explicit `\n`. Would you be ok with that?

Reply:
I'm fine with that. I generally think that automatically stripping of whitespace is a bad idea for Jedi in string literals. This has to do with a fact that it's pretty much necessary to test this kind of stuff.